handler.go

```diff
@@ -1691,35 +1704,12 @@ func (h *Handler) InputJSONDataParser(req map[string]interface{}, index *Index,
 		if _, ok := req[field.Name]; !ok {
 			continue
 		}
-		value, ok := req[columnLabel]
-		if !ok {
-			return nil, fmt.Errorf("columnLabel required")
-		}
-		colValue, ok := value.(float64)
-		if !ok {
-			return nil, fmt.Errorf("float64 require, got value:%s, type: %s", value, reflect.TypeOf(value))
-		}
 
 		// Looking into timestampFrame map and set timestamp to the whole frame
-		var timestamp string
 		for _, action := range field.Actions {
 			frame := action.Frame
-			timeField, ok := timestampFrame[action.Frame]
-			if !ok {
-				timestamp = ""
-			} else {
-				tmstamp, ok := req[timeField]
-				if !ok {
-					timestamp = ""
-				} else {
-					timestamp, ok = tmstamp.(string)
-					if !ok {
-						return nil, fmt.Errorf("set-timestamp value must be in time format: YYYY-MM-DD, having: %v", req[timeField])
-					}
-				}
-			}
-
-			bit, err := HandleAction(action, req[field.Name], uint64(colValue), timestamp)
+			timestamp := timestampFrame[action.Frame]
```

Comment:
We still need to check if action.Frame is in timestampFrame here since there's a chance that action.Frame isn't define in timestamp field and will be thrown error here